src/app/components/elements/WalletSubMenu.story.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, text } from '@storybook/addon-knobs';
+import WalletSubMenu from './WalletSubMenu';
+import { Center } from './Tooltip.story';
+import rootReducer from 'app/redux/RootReducer';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
+const store = createStore(rootReducer);
+
+storiesOf('Elements', module)
+    .addDecorator(withKnobs)
+    .addDecorator(getStory => <Provider store={store}>{getStory()}</Provider>)
+    .add('WalletSubMenu', () => (
+        <Center>
```

Comment:
should we add default styles to the storybook to just center everything so we don't have to `<Center>` in each file?

Reply:
Yes, not everything is centered. I would agree that the center wrapper should go into the storybooks config as a decorator. Like this:
https://storybook.js.org/addons/introduction/#storybook-decorators